packages/app-admin-ui/client/pages/Item/index.js

```diff
@@ -450,7 +456,7 @@ const ItemPage = ({ list, itemId, adminPath, getListByKey, toastManager }) => {
                         onUpdate={() =>
                           refetch().then(refetchedData => deserializeItem(list, refetchedData.data))
                         }
-                        toastManager={toastManager}
+                        toastManager={{ addToast }}
```

Comment:
Regarding my [earlier comment](https://github.com/keystonejs/keystone-5/pull/1544/files#r318331292):

```suggestion
                        toastManager={toastManager}
```